src/MultiDropdown.js

```diff
@@ -3,20 +3,70 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
-
-// HINT User CustomDropdown.js here
+import Select from "react-select";
+import CustomDropdown from "./CustomDropdown.js";
 
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      country: "",
+      city: ""
+    };
+    this.countryList = _.map(countriesCities.getCountries(), country => {
+      return {
+        label: country,
+        value: country
+      };
+    });
+  }
+  getCountryList(value) {
```

Comment:
Why is this called `getCountryList`? You're not getting a country list anywhere.

Reply:
changed function name